recipe/meta.yaml

```
@@ -0,0 +1,88 @@
+{% set name = "cspy" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torressa/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a586f7f8a261770bfa0c06ad994e87911bac92200c418bfd48ba5d7a4dde1734
+  patches:
+    - patches/0001-fix-export-name-for-lemon-in-findlemon.patch
+    - patches/0002-fix-use-spdlog-target.patch
+    - patches/0003-add-win-symbols.patch
+    - patches/0004-do-not-download-swig.patch
+    - patches/0005-install-the-biDirectionalCpp.dll.patch
+    - patches/0006-do-not-create-copy-dot-lib-directory-remove-origin.patch
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - '$RPATH/ld64.so.1'  # [s390x]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - patch       # [not win]
+    - m2-patch    # [win]
+  host:
+    - python
+    - pip
+    - spdlog
+    # in run_exports of spdlog
+    - fmt
+    - lemon 1.3.1
+    - swig
+    - setuptools
+    - wheel
+  run:
+    - python
+    - networkx
+    - numpy
+    - lemon
```

```suggestion
    - lemon 1.3.1
```
We also want to pin at runtime since we don't know what type of compatibility they offer. Or at least when we look at the DSOs, they say that they don't have a stable ABI (it could be intentional or not).